web/meetup-facilitator/src/Timepage/TimeHour.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function TimeHour() {
+  console.log("TimeHour rendered");
```

Comment:
Let's remove debug console logs, since we don't want to leak what's going on in the background once we're in production